package.json

```diff
@@ -45,6 +45,7 @@
     "cacache": "^11.0.2",
     "call-limit": "~1.1.0",
     "chownr": "~1.0.1",
+    "ci-info": "^1.1.3",
```

Comment:
There's still no `package-lock.json` change in this PR? We'll need that to land this.

Reply:
I remember commiting that lock file `node_modules/ci-info/package.json`. isn't that the one?

Reply:
@iarna I got it. The `ci-info` is already used as a dependecy of `update-notifier`

`update-notifier -> is-ci -> ci-info`

Reply:
ohhhhh

that makes sense.